### PR TITLE
modifications to resolve syntax_type_naming issues

### PR DESCRIPTION
**Pull request by Tee-Hui**
1. requirement file renamed from "reqirement" to "requirement"
2. change list to List for pydantic model
3. changed json_string_from_file to text_from _file and changed .json to .txt in code. otherwise file not located error will show.
4. Optional syntax changed

### CODE REVIEW

**Comment by Tee-Hui** [comment]
1. requirement file renamed from "reqirement" to "requirement"
2. change list to List for pydantic model
3. changed json_string_from_file to text_from _file and changed .json to .txt in code. otherwise file not located error will show.
4. Optional syntax changed

File: .DS_Store
Comment (ashishawasthi) [inline .DS_Store:1]: please remove .DS_Store files

File: assets/.DS_Store
Comment (ashishawasthi) [inline assets/.DS_Store:1]: please remove .DS_Store files

File: assets/prompts/.DS_Store
Comment (ashishawasthi) [inline assets/prompts/.DS_Store:1]: please remove .DS_Store files

**Review by ashishawasthi — CHANGES_REQUESTED** [review]
I've added  .DS_Store to gitignore in your branch. if you cancel this PR and crate a new one, it should be good

**Pull request closed without merging**